src/bindings/js/docs/CODESTYLE.md

```diff
@@ -1,9 +1,14 @@
 # Code Style Guide
 
+Node.js bindings contain two parts: C++ and Typescript/JavaScript.
+
 This article presents the coding standards for JavaScript and TypeScript parts of **openvino-node** package. The following rules will help maintain code quality and consistency throughout the codebase.
 
+For C++ codestyle rules, refer to [this document](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md).
```

Comment:
```suggestion
For C++ codestyle rules, refer to [OpenVINO Coding Guidelines](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md).
```

Reply:
@rkazants good remark! We will include this change into master in the next related PR (it's port of master PR, we don't want to initiate new changes loop)